week-1-ts-ecosystem/w01e01-typescript-basics/src/helpers.ts

```diff
@@ -0,0 +1,13 @@
+import {User} from "./types";
+
+function hasAddress(user: User): boolean {
+  return Boolean(user.address)
+}
+
+function hasGivenAge(requiredAge: number): (user: User) => boolean {
+  return (user) => user.age >= requiredAge
+}
+
+const isAdult = hasGivenAge(18)
+
+export {hasAddress, hasGivenAge, isAdult}
```

Comment:
Zauważyłem że Twój podział aplikacji wg. funkcjonalności jest bardzo dobry, w sensie ładnie rozbiłeś ten `main.ts` 

Jednak samo nazewnictwo plików bym poprawił. Jest dość generyczne i ogólne.
Zobacz mamy:
`helpers.ts` , `model.ts` , `types.ts`

Te nazwy "nie zdradzają" nam co możemy znaleźć w środku.
Tutaj na przykład są odpowiednie metody dotyczące walidacji usera

jeśli zobaczysz nazwę pliku np. 
`user.validators.ts`

to bez wchodzenia do pliku możesz zacząć się domyślać co tam w środku jest - no nie ?

Pomyśl o tym w ten sposób:
- co jeśli aplikacja się rozrośnie i tego wszystkiego będzie więcej ?